.travis.yml

```diff
@@ -3,13 +3,13 @@ os:
     - linux
 julia:
     - 0.4
+    - 0.5
     - nightly
 notifications:
     email: false
-sudo: false
```

Comment:
technically `sudo: false` can still help with routing to the docker infrastructure, otherwise travis might use the full VM's instead of the containers